ambry-store/src/test/java/com/github/ambry/store/StoreTestUtils.java

```diff
@@ -140,6 +144,54 @@ public void setSealedState(boolean isSealed) {
     }
   }
 
+  public static class MockAmbryReplica extends AmbryReplica {
```

Comment:
Why can't `MockReplicaId` be used here?

Reply:
`setReplicaDisabledState` in HelixParticipant checks if passed in replicaId is an instance of AmbryReplica